wp/plugin/README.md

```diff
@@ -329,6 +329,9 @@ The various options are:
 - `--wp-output-vars=var_list`. List of output variables for equivalence checking
   by `,` given the same input variables in the case of a comparative analysis.
   Defaults to `RAX,EAX` which are the 64- and 32-bit output registers for x86.
+- `--wp-gdb-filename=my_exec.gdb` output a gdb script to file `my_exec.gdb` which 
+  will fill in the . From within gdb, run `source my_exec.gdb` to set a breakpoint at the function given by `--wp-function` and fill the appropriate registers with a found counter-model.
```

Comment:
Can you explain what `.` is?

Reply:
Accidental fragmented sentence. I moved the end of that sentence to the end of the next sentence